### PR TITLE
Adjust spacing

### PR DESCRIPTION
**Pull request by ztolley**
Adjust the spacing of controls and also correct some markup to make it semantically correct, so it's easier to control it's spacing also.

### CODE REVIEW

File: app/views/research_sessions/researcher.html.erb [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-      </div>
+    <fieldset clas="fieldset" name="second-researcher">
+      <legend class="fieldset__legend">
+        What is the name of the second researcher?
+        <span class="fieldset__hint">We recommend design research always be conducted in pairs.</span>
+      </legend>
 
-      <%= render 'ok_cancel' %>
+      <%= labelled_text_field_tag :researcher_other_name, 'Full name', @research_session.researcher_other_name %>
+    </fielset>
```
Comment (rgarner) [inline app/views/research_sessions/researcher.html.erb]: Missing a 'd'?

File: app/assets/stylesheets/barnardos/_type.scss [hunk 2]
```diff
@@ -79,24 +77,24 @@ p.short-text {
...
   font-size: 18px;
   line-height: 24px;
-  margin: 0 0 24px;
+  margin: 24px 0;
 
   @media (min-width: $tablet) {
     font-size: 23px;
     line-height: 36px;
-    margin: 0 0 32px;
+    margin: 32px 0;
```
Comment (rgarner) [inline app/assets/stylesheets/barnardos/_type.scss]: Seeing a lot of multiples of `8px` in here – is there a pattern to them we can make explicit?

File: app/views/research_sessions/researcher.html.erb [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+      <legend class="fieldset__legend">What is the name of the lead researcher with the participant?</legend>
+      <%= labelled_text_field_tag :researcher_name, 'Full name', @research_session.researcher_name %>
+      <%= labelled_text_field_tag :researcher_phone, 'Telephone number', @research_session.researcher_phone %>
+      <%= labelled_text_field_tag :researcher_email, 'Email', @research_session.researcher_email %>
+    </fieldset>
 
-      <div class="second-researcher">
-        <%= labelled_text_field_tag :researcher_other_name, 'Full name', @research_session.researcher_other_name %>
-      </div>
+    <fieldset clas="fieldset" name="second-researcher">
```
Comment (rgarner) [inline app/views/research_sessions/researcher.html.erb]: missing 's' on `class`

File: app/views/research_sessions/researcher.html.erb [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
   <%= form_for @research_session, url: wizard_path do %>
-      <div class="first-researcher">
-        <%= labelled_text_field_tag :researcher_name, 'Full name', @research_session.researcher_name %>
-        <%= labelled_text_field_tag :researcher_phone, 'Telephone number', @research_session.researcher_phone %>
-        <%= labelled_text_field_tag :researcher_email, 'Email', @research_session.researcher_email %>
-      </div>
 
-      <h2 class="heading-medium">What is the name of the second researcher?</h2>
-      <span class="hint">We recommend design research always be conducted in pairs.</span>
+    <fieldset class="fieldset" name="first-researcher">
```
Comment (rgarner) [inline app/views/research_sessions/researcher.html.erb]: Fieldsets can have a `name`? This is indeed (probably good) news to me.

Comment (ztolley) [inline app/views/research_sessions/researcher.html.erb]: Yes they can, I checked

File: app/assets/stylesheets/barnardos/_type.scss (hunk 2 above)
Comment (ztolley) [inline app/assets/stylesheets/barnardos/_type.scss]: We use multiples of 8 and 16. It may be worth having a separate PR to put these in as named units
